src/airport.js

```diff
@@ -1 +1,53 @@
+class Airport {
+    #airportCapacity;
+    #planesStanding;
 
+    constructor(airportCapacity = 10, planesStanding = []) {
+        this.#airportCapacity = airportCapacity;
+        this.#planesStanding = planesStanding;
+    }
+
+    landPlane(plane) {
+        if (!plane?.id) return `Unidentified planes are not allowed at the airport`
+        if (this.fullAirport()) {
+            return `Sorry, ${plane.id}, no room for you to park!`;
+        } else if (!this.#planesStanding.includes(plane)) {
+            this.#planesStanding.push(plane);
+            return `Welcome to your destination, ${plane.id}`;
+        } else {
+            return `${plane.id} is already here`;
+        }
+    }
+
+    orderTakeOff(plane) {
+        if (this.#planesStanding.includes(plane)) {
```

Comment:
This conditional segment took me forever to develop because I was trying to use the filter method, which I still don't understand very well. I went with a slightly convoluted, but simpler version in the end and I think it looks great! It's easier to read, but the filter would have halved the number of lines.